### ISSUE TITLE
[Feature Request]: 

### ISSUE THREAD

**Issue by hallool** [AlexNijjar/The-Extractinator#39]
### Is your feature request related to a problem?

i would like to add some items that are not automaticaly included for example  yellorium from extreme reactors or some self made stuff

and maybe add some blocks that can be crushed like example compressed cobblestone

just examples

i want this for personal use dont know if this mod is free to use for modpacks

### Solution(s)

knowledge of how to change/add item drops and blocks to crush

### Describe alternatives you've considered

maybe an config .json option? may be helpful for modpacks or personal use

### Mod Version

1.20.1

### Mod Loader Version

1.20.1 47.1.3 

### Mod Loader

Forge

### Additional context

i have looked in to the jar file but could not find anything about item drops  maybe im just too lost

**Comment by ABSOLUTEMAX123** [comment]
i agree with this. theres a lot of really good not well known mods that arent included. maybe finding a way to add all existing raw ores when the mod initializes?
